free-2.md

```diff
@@ -34,36 +38,44 @@ _What might happen when a program runs? How does a program interact with the
 the user?_
 
+I imagine the program can be run on the terminal. At first, the user will be able to import a picture they already own, or input keywords to get a picture randomly from Google.
+The user might find it inconvenient that they can't see the final product of their coloring page on the terminal since this program is not interactive that way.
+
 
 ### Expressiveness
 _What should be easy to do in this language? What should be possible, but
 difficult? What should be impossible or very difficult?_
 
+It should be easy for the user to generate coloring pages with pictures they already own or pictures found on Google based on their input keywords. They will also be able to specify their own arbitrary settings such as removing all colors from the original pictures except purple.
+However, it would be impossible or very difficult for the user to take a peak on the final product before the program ends. In other words, they wouldn't be able to interact with the final product in real-time.
 
 ### Related work
 _Are there any other DSLs in this domain? If not, describe how you know there
-aren't and conjecture why not. If so, describe them and provide links. How well 
-do they address the need? Are there any particularly admirable qualities of the
+aren't and conjecture why not. If so, describe them and provide links. How well do they address the need? Are there any particularly admirable qualities of the
 language? Are there parts of the language you think could be improved?_
 
+I had no idea this existed before I just did some research, there's a website that allows users to convert photos into coloring pages, https://www.reallycolor.com/. I want my DSL to essentially do the same thing, but also provide users with different settings such as removing all colorings except a certain color.
 
 ## The Project
 This section examines whether the idea makes for a good CS 111 project.
 
 
 ### Suitability
-_If someone were to work on this project, what percentage of their time would be
-spent directly engaging in the **language** aspects of this project (e.g.,
+_If someone were to work on this project, what percentage of their time would be spent directly engaging in the **language** aspects of this project (e.g.,
 making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+Developing language aspects of the project would be time-consuming since manipulating colors and other things of photos can be hard. But I don't imagine this project needing to integrate with any APIs. 
+And the system part would be less time-consuming, since most of it would be converting the picture into a sketched veresion of it, possibly in a pdf.
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+I think this project is ambitious for me in a way that it's going to require a lot of image manipulation, which I've always found hard. 
```

Comment:
Mudd's computer vision class does a ton of image manipulation (in fact our assignment a few weeks ago basically did this).